### DISCUSSION TITLE
[Internationalization and localization] Localize store listing description

### DISCUSSION THREAD

**Discussion by pnp0a03** [0x7c13/Notepads#817] [General]
Do you have a plan to localize the store listing information? I believe that it's important for the non-native-english users. 
I've added my translation for japanese. Please use it if you required.

![image](https://user-images.githubusercontent.com/10514567/64744628-83a81f00-d53f-11e9-974d-0608b4ac2750.png)

# Notepads - モダンでスタイリッシュな、ミニマル デザインのオープンソース テキストエディター

- タブ対応のFluent デザイン
- Win32版メモ帳のような超高速起動、なのにカッコいい
- コンソール・PowerShell からコマンドラインで「Notepads」と入力して起動可能
- 複数行の手書き入力をサポート
- Markdown 書式のプレビュー機能を内蔵
- diff 機能を内蔵（ファイルの変更点プレビュー）

# Notepads とは何で、なぜ作ったのか？

自分で手を付ける以前、私は長い間Windows 10 のモダンなメモ帳アプリが出現するのを待っていました。誤解しないで欲しいのですが、Notepad++,  Atom,  VS Code,  そしてSublime は優れたテキストエディターです。私は殆どを使ったことがありますし、今後も使い続けるでしょう。しかし、それらは重く、又は見た目が古いです。私が本当に欲しいのは軽く、シンプルで、用事を素早く済ませられる、設定ファイルを素早く編集したりさっとメモを取るのに適したテキストエディターとして使える、Win32版メモ帳に取って代わる物です。そして最も重要な事は、それはとても高速で、かつ美しいものでなければなりません。

Notepads は今現在も活発に開発が続けられています。最新の情報はこちらをご参照下さい:
http://www.NotepadsApp.com

**Comment by 0x7c13** [comment]
Thanks for doing it! Very appreciated and yes, I do need these strings to be translated.

**Comment by FJduFou** [comment]
Bloc-notes - Un éditeur de texte open source moderne, élégant et minimaliste.

- Design fluide avec système de tabulation intégré.
- Incroyablement rapide, ressemble à win32 notepad.exe mais semble mieux.
- Lancez depuis la ligne de commande ou PowerShell en tapant: "notepads"
- Prise en charge de l'écriture multi-lignes.
- Visualiseur de prévisualisation de fichier intégré dans Markdown.
- Visualiseur de différences intégré (prévisualisez vos modifications).

Qu'est-ce qu'un bloc-notes et pourquoi je m'en soucie?

J'ai attendu assez longtemps pour qu'une application moderne Windows 10 Bloc-notes vienne avant de devoir le faire moi-même. Ne vous méprenez pas, Notepad ++, Atom, VS Code et Sublime sont de bons éditeurs de texte. J'ai utilisé la plupart d'entre eux et je continuerai à les utiliser à l'avenir. Cependant, ils sont soit trop lourds, soit vieux. J'ai vraiment besoin d'un remplacement de win32 notepad.exe léger, simple et propre pour m'aider à accomplir les tâches le plus rapidement possible et à l'utiliser comme éditeur de texte de redressement pour modifier rapidement les fichiers de configuration ou rédiger des notes. Plus important encore, il doit être extrêmement rapide et beau.

Notepads est encore en développement actif. Veuillez vérifier les dernières informations ici: http://www.NotepadsApp.com

**Comment by tarik02** [comment]
uk-UA (ukrainian) store text translation:
```
Notepads - сучасний, стильний текстовий редактор з відкритим кодом і мінімалістичним дизайном.

- Дизайн в стилі Fluent, вбудована система вкладок.
- Вражаюче швидкий, відчувається ніби стандартний блокнот, але виглядає краще.
- Запускається з командного рядка або PowerShell командою "notepads".
- Підтримка багаторядкового рукописного введення.
- Вбудований переглядач файлів формату Markdown.
- Вбудований переглядач різниці між файлами (зміни в файлі).

Що таке Notepads і навіщо він потрібен?

Я досить довго чекав сучасного блокнота для Windows 10 до того, як я його створив сам. Не зрозумійте мене неправильно, Notepad++, Atom, VS Code і Sublime це хороші редактори тексту. Я використовував більшість з них і продовжу використовувати їх надалі. Але, вони або занадто тяжкі, або виглядають застарілими. Мені справді потрібна заміна класичного блокнота (notepad.exe), яка відчувається такою ж легкою, простою і чистою, що допомагає мені зробити речі швидко, як можна і використовувати її як звичайний редактор щоб швидко відредагувати конфігураційні файли, або написати якісь записки. Що найважливіше, вона повинна бути вражаюче швидкою і красивою.

Notepads досі знаходиться під активною розробкою. Остання інформація може бути отримана тут: http://www.NotepadsApp.com
```

**Comment by validatedev** [comment]
My implementation of tr-TR translation:

> Notepads – Modern, şık ve açık kaynak bir metin düzenleyicisi
> -Fluent Design dizayn anlayışına uyumlu ve sekme sistemine sahip.
> -Hızlı, Windows ile öntanımlı olarak gelen Not Defteri uygulaması gibi ama daha iyisi.
> -Powershell veya Komut İstemi aracılığıyla “notepads” komutuyla çalıştırabilme imkanı.
> -Çoklu satır el yazısı desteği.
> -Öntanımlı olarak Markdown dosyalarını görüntüleyebilme imkanı.
> -Öntanımlı diff görüntüleyicisine sahip (İki dosya arasındaki farkları gösterir).
> 
> Notepads nedir ve neden önemlidir?
> Bu uygulamayı yapmadan önce modern bir Windows 10 Not Defteri uygulaması için uzun bir süre bekledim. Beni yanlış anlamayın, Notepad++, Atom, VS Code ve Sublime Text uygulamaları alanlarında kendilerini kanıtlamış metin düzenleyicileri, buna bir sözüm yok, ayrıca yukarıda belirttiğim bu uygulamaları çokça kullandım ve gelecekte de bu uygulamalardan yararlanmayı düşünüyorum ancak bunlar ya çok ağırlar ya da eski görünüyorlar. Benim asıl ihtiyacım olan şey; hafif, basit, işlerimi kısa sürede halletmemi sağlayan -bunlar konfigürasyon dosyalarını düzenlemek veya not almak olabilir- Not Defteri yerine geçebilecek bir uygulamaydı. En önemlisi de, gerçekten hızlı olmalıydı ve göze hitap etmeliydi.
> 
> Notepads hala aktif bir şekilde üzerinde çalışılan ve geliştirilen bir uygulamadır. Son gelişmelerden haberdar olmak için:
> http://www.NotepadsApp.com

**Comment by pinholucas** [comment]
**pt-BR** :)
-

Notepads - Um editor de texto moderno, estiloso, de código aberto e com design minimalista.

- Design Fluent com sistema de abas integrado.
- Incrivelmente rápido, como o notepad.exe win32, porém com melhor aparência.
- Inicia a partir do Prompt de Comando ou PowerShell ao digitar: "notepads"
- Suporte para escrita manual de múltiplas linhas.
- Visualizador de arquivos Markdown integrado.
- Comparador de diferenças integrado (Visualize suas alterações).

O que é o Notepads e por que eu me importo?

Eu esperei tempo o suficiente por um aplicativo moderno do Notepad no Windows 10 antes de decidir fazê-lo por conta própria. Não me entenda mal, Notepad++, Atom, VS Code e Sublime são bons editores de texto. Eu tenho usado a maioria deles e continuarei a usá-los. Entretanto, ou eles são muito pesados ou parecem velhos. Eu realmente preciso de um substituto para o notepad.exe win32 que seja leve, simples e minimalista para me ajudar a fazer as coisas o mais rápido o possível e usá-lo como um editor quebra galho para modificar rapidamente arquivos config ou escrever alguma anotação. Mais importante, ele tem que ser incrivelmente rápido e bonito. 

Notepads ainda está sob ativo desenvolvimento. Confira as últimas informações aqui: 
http://www.NotepadsApp.com

**Comment by fitojb** [comment]
= es =
Notepads: un editor de textos moderno, estilizado y de código abierto con un diseño minimalista.

- Diseño «Fluent» y sistema de pestañas incorporado.
- Rapidísimo; se siente como utilizar notepad.exe de Win32, pero con un mejor aspecto.
- Inícielo desde el intérprete de órdenes o PowerShell escribiendo: «notepads»
- Admite la escritura a mano multirrenglón.
- Previsualizador incorporado para archivos Markdown.
- Visor de diferencias incorporado (previsualice sus modificaciones).

¿Qué es Notepads y por qué lo he creado?

He esperado bastante tiempo a que surgiera una versión moderna del Bloc de notas de Windows 10 hasta que decidí crear una. No quisiera ser malinterpretado: Notepad++, Atom, VS Code y Sublime son editores de texto excelentes. He utilizado la mayoría de estos y seguiré haciéndolo; sin embargo, o son muy pesados o lucen anticuados. En verdad necesito un sustituto para notepad.exe Win32 que se sienta ligero, sencillo y limpio para ayudarme a realizar mis tareas lo más rápidamente posible y que pueda utilizar para modificar al instante archivos de configuración o escribir notas. Ante todo, debe ser veloz y atractivo.

Notepads aún se encuentra en desarrollo activo. Obtenga la información más reciente aquí: http://www.NotepadsApp.com
